Makefile

```diff
@@ -38,7 +38,7 @@ all-apps: $(APPS)
 bundle-%: clone-%
 	$(GOVUK_DOCKER) build $*-lite
 	$(GOVUK_DOCKER) run $*-lite rbenv install -s || ($(GOVUK_DOCKER) build --no-cache $*-lite; $(GOVUK_DOCKER) run $*-lite rbenv install -s)
-	$(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'
+	if [ -f Gemfile.lock ]; then $(GOVUK_DOCKER) run $*-lite sh -c 'gem install --conservative --no-document bundler -v $$(grep -A1 "BUNDLED WITH" Gemfile.lock | tail -1)'; fi
```

Comment:
What happens if we haven't installed bundler for a gem?

Reply:
Interesting, I'd not considered that.
I'm going to go through the rest of the repos that have a gemspec and see what the state of play is. Then I'll feed back here if there _are_ any that don't have bundler installed. I should be able to see if this step trips up any others we try to make.

If there are not, then perhaps we can have a chat about best pratice and what's good enough for now?

If there are some, then perhaps I'll take this line and abstract it off into it's own make step or bash function that can choose if we install bundler with slightly more complex logic.

Reply:
Sure, thanks. I only recently learnt that Ruby now comes with bundler pre-installed, but it would be good to verify all the gemspec projects we support actually use a sufficient version for that. If not, they could just have their own custom Ruby/bundle thing in their Makefile, since they're intent on being special (or we could just upgrade Ruby!).

Reply:
Ah interesting!
Looks like bundler has been in standard lib [since ruby 2.6.0](https://ruby-doc.org/core-2.6/doc/standard_library_rdoc.html).
I'll take a quick look now for anything we've got in GOV.UK docker with a gemspec, and see if it's >2.6.0.

Reply:
so I think i've found five gems that are supported by `govuk-docker`.
From my `~/govuk` if i run:
```
cat govspeak/.ruby-version && \
cat plek/.ruby-version && \
cat gds-api-adapters/.ruby-version && \
cat govuk_publishing_components/.ruby-version && \
cat govuk_app_config/.ruby-version
```
i get:

```
2.6.3
2.6.6
2.6.6
2.6.6
2.6.5
```

So currently I believe all supported gems are a version above 2.6.x and so should come with bundler in standard lib! :tada:

Reply:
So for the moment you happy to approve with this if statement?
If we have a need for more gems in future, or to support earlier ruby versions I'm happy to look at something more elegant?

Reply:
Nice, thanks for investigating! I'm glad we can keep it simple.